pyproject.toml

```diff
@@ -14,7 +14,10 @@ license = {text = "Apache Software License 2.0"}
 classifiers = [
     "Programming Language :: Python :: 3",
 ]
-dependencies = []
+dependencies = [
+  'armonik',
+  'numpy',
```

Comment:
if you want to use the example, matplotlib is missing. It can be in another environment in the pyproject.toml